src/action.rs

```diff
@@ -109,32 +109,66 @@ pub fn open_file(
     options: FileDialogOptions,
     file_info_action: impl Fn(Option<FileInfo>) + 'static,
 ) {
-    let send = create_ext_action(Scope::new(), move |path: Option<PathBuf>| {
-        file_info_action(path.map(|path| FileInfo { path, format: None }))
-    });
+    let send = create_ext_action(
+        Scope::new(),
+        move |(path, paths): (Option<PathBuf>, Option<Vec<PathBuf>>)| {
```

Comment:
I needed to juggle the two different types you get with `multi_selection` enabled. This was the best I could come up with.